staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/errors.go

```diff
@@ -47,9 +47,9 @@ func Forbidden(ctx request.Context, attributes authorizer.Attributes, w http.Res
 
 	var errMsg string
 	if len(reason) == 0 {
-		errMsg = fmt.Sprintf("%s", msg)
+		errMsg = fmt.Sprintf("%s.", msg)
```

Comment:
I wouldn't blindly try to coerce these into sentences... why not omit the `.` entirely?

Reply:
Done.